.github/workflows/publish-provider-package.yaml

```diff
@@ -0,0 +1,25 @@
+name: Publish Provider Package
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version string to use while publishing the package (e.g. v1.0.0-alpha.1)"
+        default: ""
+        required: false
+      go-version:
+        description: "Go version to use if building needs to be done"
+        default: "1.22"
+        required: false
+
+jobs:
+  publish-provider-package:
+    uses: crossplane-contrib/provider-workflows/.github/workflows/publish-provider-non-family.yml@main
+    with:
+      repository: provider-upjet-digitalocean
+      version: ${{ github.event.inputs.version }}
+      go-version: ${{ github.event.inputs.go-version }}
+      cleanup-disk: true
+    secrets:
+      GHCR_PAT: ${{ secrets.GITHUB_TOKEN }}
+      XPKG_UPBOUND_TOKEN: ${{ secrets.XPKG_UPBOUND_TOKEN }}
```

Comment:
@jbw976 would this be an org-wide secret that we could use?

Reply:
the secrets should be per repo instead of org level, so limit blast radius. Looks like this repo already has a secret defined as `XPKG_TOKEN`, so you can just use that here instead! e.g.:
```
      XPKG_UPBOUND_TOKEN: ${{ secrets.XPKG_TOKEN }}
```